fapi-client/src/main/scala/com/gu/facia/api/utils/ResolvedMetaData.scala

```diff
@@ -21,6 +21,18 @@ object ResolvedMetaData {
   def isVideoForContent(content: Content): Boolean =
     content.tags.exists(_.id == Video)
 
+  def isVideoAtom(content: Content): Boolean = {
+    val maybeAtom = for {
+      videoContentType <- content.tags.find(_.id == Video)
+      blocks <- content.blocks
+      main <- blocks.main
+      atomElement <- main.elements.find(e => e.`type` == ElementType.Contentatom)
+    } yield atomElement
```

Comment:
This `for` comprehension is tricky to parse if you aren't reading it carefully.

Could you either add the type of `Option[List[Option[Element]]` or change it around to make it easier to read? I think it would be clearer what you are trying to do by using `exists` and then not having to use the `isDefined` on the big nested type.